mapping/requirements_officejs.json

```diff
@@ -5352,7 +5352,7 @@
 						},
 						{
 							"name": "AddinCommands",
-							"apiVersion": "1.1",
+							"apiVersion": "1.3",
```

Comment:
Please let me know what you find out regarding version and whether a new block must  be added instead such as 1.1, 1.2? and finally 1.3.